fe1-web/components/eventList/LaoProperties.tsx

```diff
@@ -10,21 +10,28 @@ import { useSelector } from 'react-redux';
 import ParagraphBlock from 'components/ParagraphBlock';
 import { Lao } from 'model/objects';
 import { makeCurrentLao } from 'store/reducers';
+import PropTypes from 'prop-types';
+import { ConnectToLao } from 'model/objects/ConnectToLao';
 import QRCode from '../QRCode';
 
-function renderProperties(lao: Lao) {
+function renderProperties(lao: Lao, url: string) {
   const creationDateString = lao.creation.timestampToString();
+  const obj = new ConnectToLao({
+    server: url,
+    lao: lao.id.toString(),
+  });
+  const connectToLao = ConnectToLao.fromJson(obj);
```

Comment:
why would we do a `ConnectToLao.fromJson` here if we have already created the `ConnectToLao` object?